core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -473,7 +474,7 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
     val rawCwd = System.getProperty("user.dir")
     val cwd = if (Utils.isWindows) s"/$rawCwd".replace("\\", "/") else rawCwd
     assertResolves("hdfs:/root/spark.jar", "hdfs:/root/spark.jar")
-    assertResolves("hdfs:///root/spark.jar#app.jar", "hdfs:/root/spark.jar#app.jar")
+    assertResolves("hdfs:///root/spark.jar#app.jar", "hdfs:///root/spark.jar#app.jar")
```

Comment:
what is the `before: String` targets in `assertResolves(`before: String`, after: String)` ? 
It seems to me that the different values of `before: String` test more different cases of `Utils.resolveURI`. if we do not  use `before: String` to check, why add the params of `before: String`